docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
-# Localization component design
+LOCALIZATION COMPONENT DESIGN DOC
 
-!!! warning
+## Abstract
 
-    Under Construction
+## 1. Requirements
+
+Localization aims to estimate vehicle pose, velocity, and acceleration.
+
+Goals:
+
+- Proposition of a system that can estimate the vehicle pose, velocity, and acceleration as long as possible
+- Proposition of a system that can diagnose the stability of estimation and send a warning message to the error-monitoring system if the estimation result is unreliable
+- Vehicle localization function can adequately work with various sensor configurations
```

Comment:
I am not 100% confident about this suggestion. I do note that the original phrasing of the third sentence does not fit with the first two goals, so some additional thinking may be required. 

```suggestion
- Propose a system that can estimate vehicle pose, velocity, and acceleration for as long as possible
- Propose a system that can diagnose the stability of estimation and send a warning message to the error-monitoring system if the estimation result is unreliable
- Design a vehicle localization function can adequately work with various sensor configurations
```

Reply:
Oops. Thank you for that. I have updated the suggestion accordingly.

Reply:
I also removed the word "adequately" from the third point, since it has a negative feeling to it and I do not think it is necessary to include in the sentence.